script/clean-up-class

```diff
@@ -5,44 +5,40 @@
 # shellcheck disable=SC1091
 source "$HOME/.trainingmanualrc"
 
+# shellcheck source=script/shared_functions
+source ./shared_functions
+
 #################################################################
 # NOTE: You must have a personal access token (PAT)             #
 # saved to your environment variables to use this script.       #
 # We recommend a dedicated service account (e.g. githubteacher) #
 #################################################################
 
-# GLOBALS
-COLLAB_REPO=$1
-
-function GetNames() {
-  # Get the collabs
-  IFS=" " read -ra COLLABS <<<"$(
-    curl -s -S -u "$TOKEN_OWNER:$TEACHER_PAT" -X GET "https://$INSTANCE_URL/repos/$CLASS_ORG/$COLLAB_REPO/collaborators?affiliation=direct&per_page=100" | jq -r 'map(.login) | unique | @sh' | tr -d \'
-  )" >>log.out 2>&1
+# shell variables
+collab_repo=$1
+org_url="https://$ROOT_URL/$CLASS_ORG"
+org_repos_endpoint="https://$INSTANCE_URL/repos/$CLASS_ORG"
 
-  # Clean up conflict repos based on collaborators in the inital class repo
-  for USER_NAME in "${COLLABS[@]}"; do
+function delete_repos() {
+  for username in "${collaborators[@]:?}"; do
     # Clean up conflict and games repos for the user
-    CleanUpRepo "conflict-practice-$USER_NAME"
-    CleanUpRepo "github-games-$USER_NAME"
+    delete_repo "conflict-practice" "$username"
+    delete_repo "github-games" "$username"
   done
 }
 
-function CleanUpRepo() {
-  # Get the Repo
-  REPO=$1
+function delete_repo() {
```

Comment:
```suggestion
delete_repo() {
```

I recommend removing `function` here, as it reduces compatibility with older versions of bash.

Source: https://github.com/dylanaraps/pure-bash-bible#function-declaration